SpatialGDK/Source/Public/Schema/UnrealMetadata.h

```diff
@@ -43,12 +44,20 @@ struct UnrealMetadata : Component
 
 		AddStringToSchema(ComponentObject, 1, StaticPath);
 		AddStringToSchema(ComponentObject, 2, OwnerWorkerAttribute);
+		AddStringToSchema(ComponentObject, 3, ClassPath);
 
 		return Data;
 	}
 
+	FORCEINLINE UClass* GetNativeEntityClass()
+	{
+		UClass* Class = FindObject<UClass>(ANY_PACKAGE, *ClassPath);
+		return Class->IsChildOf<AActor>() ? Class : nullptr;
```

Comment:
How are you guaranteed that `Class` itself won't be `nullptr`?

Reply:
UnrealMetadata should only be filled with strings that come from a class's path. I'll add an `if` just in case.